wvd-sh/WVD scaling script/basicScale-MSI.ps1

```diff
@@ -355,19 +345,28 @@ if ($hostpoolInfo.LoadBalancerType -eq "DepthFirst") {
                     if ($sessionHost.Sessions -ge $SessionhostLimit) {
                         foreach ($sHost in $getHosts) {
                             if ($sHost.Status -eq "Available" -and $sHost.Sessions -eq 0) { break }
```

Comment:
What happens if session host is "Available" but has more than "0" sessions?

Reply:
Thank you Christian, we have updated accordingly

Reply:
If reached the more than "0" sessions script will automatically spin up the one more VM.